modyn/config/schema/pipeline/trigger/drift/evidently.py

```diff
@@ -5,39 +5,52 @@
 from modyn.config.schema.pipeline.trigger.drift.metric import BaseMetric
 
 
-class _EvidentlyBaseDriftMetric(BaseMetric):
+class _EvidentlyPcaDriftMetric(BaseMetric):
     num_pca_component: int | None = Field(None)
 
 
-class EvidentlyModelDriftMetric(_EvidentlyBaseDriftMetric):
+class EvidentlyModelDriftMetric(_EvidentlyPcaDriftMetric):
     id: Literal["EvidentlyModelDriftMetric"] = Field("EvidentlyModelDriftMetric")
     bootstrap: bool = Field(False)
-    quantile_probability: float = 0.95
-    threshold: float = Field(0.55)
+    quantile_probability: float = Field(
+        0.95, description="Threshold for the evidently api, unused in modyn. Usage discourged."
```

Comment:
Copy&paste description from threshold?

Reply:
no the quantile probability is also a threshold, but in the probability space